controller/server/view/localization_page.ml

```diff
@@ -0,0 +1,99 @@
+open Tyxml.Html
+
+type generic_form_parameters =
+  { action_url: string
+  ; legend: string
+  ; select_name: string
+  ; has_value: bool
+  ; no_value_msg: string
```

Comment:
There is a small rendering difference, I think it’s better in case the placeholder is hidden from the select list. But it also adds a minor inconsistency depending if we have already selected or not a value, the placeholder will or will not be showed in the select panel.

Before:

https://user-images.githubusercontent.com/6768842/110754420-19f9f580-8248-11eb-9026-e14c81908040.mp4

After:

https://user-images.githubusercontent.com/6768842/110754430-1cf4e600-8248-11eb-9e1d-b4fa5fcabefc.mp4

https://github.com/dividat/playos/pull/64/commits/a4b0e16ecf13118fde2610b05c96136aac70e159

Reply:
:+1: